src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -230,6 +264,27 @@ public function thereShouldBeNewCatalogPromotionOnTheList(int $amount = 0): void
         Assert::count($this->responseChecker->getCollection($this->client->index()), $amount);
     }
 
+    /**
+     * @Then /^it should have "([^"]+)" discount$/
```

Comment:
this step does not suggest that it will be performed only on the first item from the collection. I would expect the same behaviour as in the step below for this one

Reply:
Indeed - I assume it's inspired by `it should apply to "PHP T-Shirt" variant and "Kotlin T-Shirt" variant` step. However, it makes sense in the scenario 😄 So maybe I would leave it for now a try to refactor it in the separate PR?